src/views/partials/head.ejs

```diff
@@ -25,10 +25,10 @@
     <link rel="stylesheet" href="<%= css('application.css') %>">
     <link rel="stylesheet" href="<%= css('dsfr.css') %>">
     <script type="module" src="<%= js('app.js') %>"></script>
-    <!-- Matomo -->
+    <!-- Matomo Tag Manager -->
     <script type="module" src="<%= js('matomo.js') %>"></script>
     <!-- Crisp -->
     <script type="module" src="<%= js('crisp.js') %>"></script>
     <noscript><p><img src="https://stats.data.gouv.fr/matomo.php?idsite=85&amp;rec=1" style="border:0;" alt="" /></p></noscript>
-    <!-- End Matomo Code -->
```

Comment:
**question**: Why the crisp comment tag is in the middle of the Matomo Tag Manager comment tag ?